src/plots/cartesian/axes.js

```diff
@@ -1799,12 +1811,18 @@ function numFormat(v, ax, fmtoverride, hover) {
         // 0 is just 0, but may get exponent if it's the last tick
         v = '0';
         isNeg = false;
-    }
-    else {
+    } else {
```

Comment:
@mdtusz good point. If you're going to put a comment in-between a one-line `if()` and a one-line `else`, adding `{ }` makes sense.